pulsar/consumer.go

```diff
@@ -182,6 +182,15 @@ type ConsumerOptions struct {
 	// > Notice: the NackBackoffPolicy will not work with `consumer.NackID(MessageID)`
 	// > because we are not able to get the redeliveryCount from the message ID.
 	NackBackoffPolicy NackBackoffPolicy
+
+	// startMessageID internally used by multitopic-reader
+	startMessageID MessageID
```

Comment:
The ConsumerOptions is a public interface so I don't think we should add package private fields here. Let's remove these.